types.d.ts

```diff
@@ -0,0 +1,41 @@
+declare module 'geojson2h3' {
+    import {Feature, FeatureCollection, GeoJsonProperties} from 'geojson';
+
+    /**
+     * Convert a GeoJSON feature to a set of hexagons. Only hexagons whose centers
+     * fall within the feature will be included. Note that conversion from GeoJSON
+     * is lossy; the resulting hexagon set only approximately describes the original
+     * shape, at a level of precision determined by the hexagon resolution.
+     */
+    export function featureToH3Set(feature: Feature, resolution: number): string[];
+
+    /**
+     * Convert a single H3 hexagon to a GeoJSON `Polygon` feature
+     */
+    export function h3ToFeature(h3Index: string, properties: GeoJsonProperties): Feature;
```

Comment:
I think the underlying H3-JS was changed to h3Index being a union type accepting either string or int arrays. Do we want to expose that through here?

Reply:
I'll leave this for a subsequent PR, as I need to review that there are no code changes needed. Should be fine AFAIK. There's a little concern about requiring a later version of `h3-js` (ideally this would be pretty liberal in what sibling package it can support), but that's probably fine.